From 3c4d8b0dcfe4912bd3b9b09484a82bb6f7f6e144 Mon Sep 17 00:00:00 2001
From: John Axel Eriksson <john@insane.se>
Date: Mon, 8 Oct 2018 23:12:22 +0200
Subject: [PATCH] Add fix

---
 backend/drm/atomic.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/backend/drm/atomic.c b/backend/drm/atomic.c
index ad80f2fd..6388cd4a 100644
--- a/backend/drm/atomic.c
+++ b/backend/drm/atomic.c
@@ -211,7 +211,7 @@ static bool atomic_crtc_set_gamma(struct wlr_drm_backend *drm,
 	// (can happen on older intel gpu's that support gamma but not degamma)
 	// TEMP: This is broken on AMDGPU. Always fallback to legacy until they get
 	// it fixed. Ref https://bugs.freedesktop.org/show_bug.cgi?id=107459
-	if (crtc->props.gamma_lut == 0 || true) {
+	if (crtc->props.gamma_lut == 0) {
 		return legacy_iface.crtc_set_gamma(drm, crtc, size, r, g, b);
 	}
 
-- 
2.19.0

